addons/l10n_gcc_invoice/models/product.py

```diff
@@ -0,0 +1,24 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from re import sub
```

Comment:
This is very subjective, but I would suggest importing the `re` module and then call `re.sub()`.
It makes it easier to understand / reuse (if we ever need other methods from `re`).
But feel free to ignore :upside_down_face: